text_extensions_for_pandas/array/char_span.py

```diff
@@ -69,8 +72,11 @@ def __init__(self, text: str, begin: int, end: int):
         self._end = end
 
     def __repr__(self) -> str:
-        return "[{}, {}): '{}'".format(self.begin, self.end,
-                                       textwrap.shorten(self.covered_text, 80))
+        if self.begin == CharSpan.NULL_OFFSET_VALUE:
+            return "None"
+        else:
+            return "[{}, {}): '{}'".format(self.begin, self.end,
+                                           textwrap.shorten(self.covered_text, 80))
```

Comment:
@frreiss  this was causing a problem when `begin` as CharSpan.NULL_OFFSET_VALUE which makes `self.covered_text` None and causes textwrap.shorten to crash. Let me know if you think it should print something other than "None" for that case.

Reply:
I'd say it's best here to synchronize with how Pandas represents missing values. The [docs](https://pandas.pydata.org/pandas-docs/stable/user_guide/missing_data.html) seem to point to "NA" being the standard representation for missing values in new code.

Reply:
Yeah, I agree. "NA" will be better.